### PR TITLE
Feat/integrate

### PR DESCRIPTION
**Pull request by BharathmuDev**
connect auth pages to backend

### CODE REVIEW

File: client/pages/auth/forgot.js [hunk 1]
```diff
@@ -20,17 +22,41 @@ const ForgotPassword = () => {
...
+        });
+        return;
+      }
+
+      const forgotHandler = await axios({
+        method: "post",
+        url: "/auth/forgot",
+        data: { payload: forgotHandler },
+      });
+      console.log(forgotHandler);
```
Comment (pavankpdev) [inline client/pages/auth/forgot.js]: Remove log

File: client/pages/auth/forgot.js [hunk 2]
```diff
@@ -20,17 +22,41 @@ const ForgotPassword = () => {
...
+
+      localStorage.setItem(
```
Comment (pavankpdev) [inline client/pages/auth/forgot.js]: Do not store forgot password token in localstorage

File: client/.env [hunk 3]
```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_NODE_API_URL=http://api.peerride.tech
```
Comment (pavankpdev) [inline client/.env]: Remove ENV

File: client/pages/auth/login.js [hunk 4]
```diff
@@ -26,27 +30,51 @@ const Login = () => {
...
+      }
+
+      const loginHandler = await axios({
+        method: "POST",
+        url: "/auth/login",
+        data: { payload: loginInput },
       });
-      return;
-    } else if (!loginInput.password) {
+      console.log(loginHandler);
```
Comment (pavankpdev) [inline client/pages/auth/login.js]: Remove logs

File: client/pages/auth/login.js [hunk 5]
```diff
@@ -26,27 +30,51 @@ const Login = () => {
...
+
+      localStorage.setItem(
+        "pride",
+        JSON.stringify({ token: loginHandler.data.token })
+      );
```
Comment (pavankpdev) [inline client/pages/auth/login.js:65]: Redirect to dashbord

File: client/pages/auth/register.js [hunk 6]
```diff
@@ -15,6 +15,8 @@ import {
 import { useState } from "react";
 import { useRouter } from "next/router";
 
+import axios from "../../config/axios";
+
 const Login = () => {
```
Comment (pavankpdev) [inline client/pages/auth/register.js]: update function name

File: client/pages/auth/register.js [hunk 7]
```diff
@@ -32,37 +34,61 @@ const Login = () => {
...
+      }
+
+      const registerHandler = await axios({
+        method: "POST",
+        url: "/auth/register",
+        data: { payload: registerInput },
       });
-      return;
-    } else if (!registerInput.password) {
+      console.log(registerHandler);
```
Comment (pavankpdev) [inline client/pages/auth/register.js]: Remove Logs

File: client/pages/auth/register.js [hunk 8]
```diff
@@ -32,37 +34,61 @@ const Login = () => {
...
+
+      localStorage.setItem(
+        "pride",
+        JSON.stringify({ token: registerHandler.data.token })
+      );
```
Comment (pavankpdev) [inline client/pages/auth/register.js:81]: Add redirect to dashboard

**Review by pavankpdev — CHANGES_REQUESTED** [review]

File: client/pages/auth/register.js [hunk 9]
```diff
@@ -6,16 +6,18 @@ import {
...
   useToast,
 } from "@chakra-ui/react";
 import { useState } from "react";
 import { useRouter } from "next/router";
+import Link from "next/link"
 
-const Login = () => {
+import axios from "../../config/axios";
+
+const register = () => {
```
Comment (pavan38kp) [inline client/pages/auth/register.js]: Capitalize function name

**Review by pavan38kp — CHANGES_REQUESTED** [review]

**Review by pavankpdev — APPROVED** [review]

**Pull request merged**